content/blog/AML_projection.md

```diff
@@ -0,0 +1,68 @@
++++
+draft = false
+type = "blog"
+image = "/blog_img/2019-06-14/"
+thumbImage = "/blog_img/2019-06-14/"
+date = "2019-06-10"
+title = "Projecting a New Set of Data onto t-SNE Without Reexecuting It"
+hardLineBreak = true 
+categories = ["Apply Domain", "tSNE", "projection", "chemotherapy"]
+joinLines = false
+author = "Iva Černoša"
+shortExcerpt = "We project single cell data from an AML patient undergoing chemotherapy onto a t-SNE of a healthy individual to analyse cell population changes in the course of treatment." 
+longExcerpt = "We analyse cell population changes in the course of chemotherapy by recreating a part of a study from Galen et al. (Cell, 2019) and projecting single cell data set from an AML patient undergoing treatment onto a t-SNE of a healthy individual." 
++++
+
+<!-- dodaj cover slike, spremeni datum na 14-->
+Batch effect can be a real nuisance when trying to project a new dataset onto an already existing t-SNE projection. With today’s blog we will show how to bypass it in scOrange by projecting single cell data set from an AML patient onto a t-SNE clustered according to cell type we have generated in our previous <a href= ”https://singlecell.biolab.si/blog/aml_identification/”> blog</a>. 
```

Comment:
"Batch effects can be a real nuisance when producing t-SNE plots of multiple data sets. "